client/js/utils.js

```diff
@@ -97,6 +98,24 @@ function toggleNotificationMarkers(newState) {
 	viewport.toggleClass("notified", newState);
 }
 
+function togglePasswordField(elem) {
+	$(elem).on("click", function() {
+		const $this = $(this);
+		const input = $this.closest("div").find("input");
+
+		input.attr("type") === "password" ? input.attr("type", "text") : input.attr("type", "password");
```

Comment:
`input.attr("type", input.attr("type") === "password" ? "text" : "password");` is shorter and reads better.

Reply:
@c-ciobanu, mind doing this?

Reply:
Sure, will commit this along with the changes on the `<i>` when @xPaw will express his opinion on what to use.